src/models/features.ts

```diff
@@ -0,0 +1,48 @@
+import {IconNames} from './icons';
+
+export type FeatureId = 'explore' | 'edit' | 'validate' | 'publish';
+
+export type Feature = {
+  id: FeatureId;
+  icon: IconNames;
+  name: string;
+  description: string;
+  learnMoreUrl: string;
+  CTA: string;
```

Comment:
I would rename this to `callToAction` , that's how we use it in another component (ReleaseNotes) and I think it's easier to understand